tests/wpunit/TEC/Tests/Events/Admin/Notice/RESTAPITest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * REST API Notice test.
+ */
+
+declare( strict_types=1 );
+
+namespace TEC\Tests\Events\Admin\Notice;
+
+use Closure;
+use Codeception\TestCase\WPTestCase;
+use TEC\Events\Admin\Notice\Rest_Api;
+use WP_Error;
+
+/**
+ * Class RESTAPITest
+ *
+ * @since TBD
+ */
+class RESTAPITest extends WPTestCase {
+
+	function test_response_blocked_from_wp_error() {
```

Comment:
In general i prefer the method named `it_should_do_that` or `should_do_that` and having `@test` as an annotation